v2/pkg/engine/datasource/graphql_datasource/graphql_subscription_client.go

```diff
@@ -29,6 +29,26 @@ import (
 
 const ackWaitTimeout = 30 * time.Second
 
+type epollState struct {
+	// connections is a map of fd -> connection to keep track of all active connections
```

Comment:
fd?

Reply:
file descriptor

Reply:
Am I the exception that it's not clear? Is it an acronym most know?

Reply:
yes, in the domain